EIPS/eip-lockable-tokens.md

```diff
@@ -0,0 +1,134 @@
+    /// @param      locker_ Who are lock the tokens
+    /// @param      amount_ The tokens summ
+    /// @return     `true` if the unlock was successful
+    function unlockFrom(address locker_, uint256 amount_) public returns (bool success);
+
+    /// @notice     Allows `locker_` to lock the amount of the tokens multiple times, up to the `value_` amount. 
+    /// @notice     If this function is called again it overwrites the current allowance with `value_`
+    /// @param      locker_ Who are lock the tokens
+    /// @param      value_ The possible locked tokens amount
+    /// @return     `true` if the approve was successful
+    function approveLock(address locker_, uint256 value_) public returns (bool success);
+
+    /// @notice     Allows `owner_` to unlock it's tokens multiple times, up to the `value_` amount.
+    /// @notice     If this function is called again it overwrites the current allowance with `value_`
+    /// @param      owner_ Who are lock the tokens
+    /// @param      value_ The possible locked tokens amount
+    /// @return     `true` if the approve was successful
+    function approveUnlock(address owner_, uint256 value_) public returns (bool success);
+
+    /// @notice     Returns the amount which `owner_` is still allowed to unlock from `locker_`
+    /// @param      owner_ The tokens owner
+    /// @param      locker_ The address locked the tokens
+    /// @return     The amount of the tokens, allowed to unlock in favor of the selected address
+    function lockedAllowance(address owner_, address locker_) public view returns (uint256 remaining);
+
+    /// @notice     Returns the amount which `locker_` is still allowed to lock from `owner_`.
+    /// @param      owner_ The tokens owner
+    /// @param      locker_ The address locked the tokens
+    /// @return     The amount of the tokens, allowed to lock in favor of the selected address
+    function unlockedAllowance(address owner_, address locker_) public view returns (uint256 remaining);
+
+    /// @notice     Collect, i.e. transfer tokens from the `owner_` account to the `locker_` in case
+    /// @param      owner_ The tokens owner
+    /// @param      locker_ The address locked the tokens
+    /// @param      amount_ The tokens summ
+    /// @return     `true` if the transfer was successful
+    function collect(address owner_, address locker_, uint256 amount_) public view (bool success);
+}
+```
+
+## Rationale
+
+This EIP's goal, as mentioned in the abstract, is to have a simple interface for serving token lock. Here are a few design decisions and why they were made:
+
+- Simple lock/unlock mechanism
+  - Unlike standard `transfer` / `transferTo` it allow to leave pledged tokens on the owner's account.
+  - Possibility to change the locked amount in selected limit.
+- Forced collateral
+  - Possibility to collect the pledged tokens and to clear the loan.
```

Comment:
The Rationale section should explain individual technical choices made within this proposal itself. This content reads more like it belongs in the Motivation section.